app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -49,6 +50,13 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         revealInitialPrompt()
       })
 
+      this.$pageIsNotUsefulButton.on('click', function (e) {
+        if (window.GOVUK.cookie('_ga') === null) {
```

Comment:
This won't catch if the cookie value has been set to blank (which is what happens when someone opts out of Google Analytics cookies via https://www.gov.uk/help/cookies )

Reply:
👍